script.js

```diff
@@ -0,0 +1,112 @@
+const minSearch = (firstNum, secondNum, thirdNum, fourthNum) => {
+  if (typeof (firstNum && secondNum && thirdNum && fourthNum) === "string") {
+    firstNum = Number(firstNum);
```

Comment:
It happens a lot in your code

Check [here](https://blog.sapegin.me/all/avoid-mutation/)